source/apiVolontaria/volunteer/views.py

```diff
@@ -278,3 +279,127 @@ def delete(self, request, *args, **kwargs):
             'detail': "You are not authorized to delete an event.",
         }
         return Response(content, status=status.HTTP_403_FORBIDDEN)
+
+
+class Participations(generics.ListCreateAPIView):
+
+    """
+
+    This view handles the link between User and Events (participations)
+
+    get:
+    Return a list of all the existing Participations.
+
+    post:
+    Create a new Participation.
+
+    """
+
+    serializer_class = serializers.ParticipationBasicSerializer
+
+    def get_queryset(self):
+        if self.request.user.is_superuser:
+            return models.Participation.objects.all()
+        elif self.request.user.has_perm('volunteer.add_participation'):
+            queryset = models.Participation.objects.all()
+
+            list_exclude = list()
+            for participation in queryset:
+                if participation.user != self.request.user:
+                    list_exclude.append(participation)
+
+            queryset = queryset.\
+                exclude(
+                    pk__in=[participation.pk for participation in list_exclude]
+                )
+
+            return queryset
```

Comment:
We can't just do `models.Participation.objects.filter(user=self.request.user)` ?